projects/aca-testing-shared/src/components/toolbar/toolbar.ts

```diff
@@ -92,7 +92,14 @@ export class Toolbar extends Component {
     await this.menu.waitForMenuToOpen();
   }
 
-  async closeMoreMenu() {
+  async closeMoreMenu(): Promise<void> {
```

Comment:
you are hiding a real bug with this. The menu does not close when pressing Escape. This is a big Accessibility issue and needs to be fixed.

Reply:
We are raising a regression for this issue. I will provide the ID to you once it's created

Reply:
Regression created: https://issues.alfresco.com/jira/browse/ACA-3679